pkgs/applications/office/paperless-ngx/default.nix

```diff
@@ -46,14 +54,26 @@ let
     pname = "paperless-ngx-frontend";
     inherit version src;
 
-    npmDepsHash = "sha256-uDaZ7j7IDgKy7wCWND2xzR1qHwUtdyjR4eyIAVy01dM=";
+    npmDepsHash = "sha256-K7wTYGGwEhPoXdRD+4swhSlMH0iem6YkF0tjnVHh7K8=";
 
     nativeBuildInputs = [
       python3
     ] ++ lib.optionals stdenv.isDarwin [
       xcbuild
     ];
 
+    buildInputs = [
+      cairo
+      giflib
+      libjpeg
+      libpng
+      librsvg
+      pango
+      pkg-config
+    ] ++ lib.optionals stdenv.isDarwin [
+      mupdf
+    ];
```

Comment:
This is probably because of https://github.com/paperless-ngx/paperless-ngx/commit/826322b6105c45c103f0da8d482121ecb3ff50d2 and the new canvas dependency

Reply:
Confirmed!